src/tests/sqlsmith/src/runner.rs

```diff
@@ -37,13 +55,31 @@ pub async fn run(client: &tokio_postgres::Client, testdata: &str, count: usize)
     drop_tables(&mviews, testdata, client).await;
 }
 
+async fn populate_tables<R: Rng>(
+    client: &tokio_postgres::Client,
+    rng: &mut R,
+    base_tables: Vec<Table>,
+    row_count: usize,
+) {
+    let inserts = insert_sql_gen(rng, base_tables, row_count);
+    tracing::info!("Executing: {}", &inserts);
+    client.query(&inserts, &[]).await.unwrap();
+}
+
 /// Sanity checks for sqlsmith
 async fn test_sqlsmith<R: Rng>(
     client: &tokio_postgres::Client,
     rng: &mut R,
     tables: Vec<Table>,
     setup_sql: &str,
+    base_tables: Vec<Table>,
+    row_count: usize,
 ) {
+    // Test inserted rows should be at least 50% population count,
```

Comment:
Is this because that <=50% inserted rows are duplicated in PK?

Reply:
Yes, otherwise it will affect effectiveness of sqlsmith